alica_tests/src/test/test_task_assignment.cpp

```diff
@@ -82,7 +83,8 @@ TEST_F(TaskAssignmentTest, constructTaskAssignment)
     ae->editRoleAssignment().tick();
 
     // fake inform the team observer about roles of none existing robots
-    alica::RunningPlan* rp = new RunningPlan(ae, ae->getPlanRepository().getPlans().find(1407152758497), nullptr);
+    alica::AlicaDefaultLogger logger(alica::Verbosity::DEBUG, "nase");
+    alica::RunningPlan* rp = new RunningPlan(ae, ae->getPlanRepository().getPlans().find(1407152758497), nullptr, logger);
```

Comment:
logger argument not required -> I think you need to fix the tests?

Reply:
all test changes are now in https://github.com/rapyuta-robotics/alica/pull/302, the logger is not passed to the RP anymore